desk/app/channels.hoon

```diff
@@ -872,8 +938,13 @@
   ++  ca-join
     |=  [n=nest:c group=flag:g]
     =.  nest  n
-    ?<  (~(has by v-channels) nest)
     ?>  |(=(p.group src.bowl) from-self)
+    ?:  (~(has by v-channels) nest)
+      ::  we should already be in, but make sure our subscriptions still exist
+      ::  just in case
+      ::
+      =.  channel  (~(got by v-channels) nest)
+      (ca-safe-sub |)
```

Comment:
this is a good change to join. just maybe something vestigial here:

```suggestion
    ?:  (~(has by v-channels) nest)
      ::  we should already be in, but make sure our subscriptions still exist
      ::  just in case
      ::
      (ca-safe-sub |)
```

Reply:
Not vestigial, `+ca-join` gets called directly from the core, instead of initializing the core through `+ca-abed` first. This is also why this arm does the `=.  nest` a couple lines above.